<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>269</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <widget class="QScrollArea" name="scroll_dates">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="dates_widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>397</width>
         <height>63</height>
        </rect>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>401</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="dates_layout"/>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scroll_times">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="times_widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>397</width>
         <height>193</height>
        </rect>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>401</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="times_layout">
         <item>
          <layout class="QGridLayout" name="times_grid"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
